ibm/mas_devops/roles/suite_app_config/vars/defaultspecs/optimizer.yml

```diff
@@ -0,0 +1,3 @@
+---
+# Default application spec for Optimizer
+mas_app_ws_spec:
```

Comment:
This is not valid  you only have half of the assignment.  need to assign empty object to this: `mas_app_ws_spec: {}`